src/test/java/integration/Nextflow.java

```diff
@@ -0,0 +1,29 @@
+package integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import harness.TestBashScript;
+import harness.TestUsers;
+import harness.baseclasses.ClearContextIntegration;
+import java.io.IOException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+public class Nextflow extends ClearContextIntegration {
+
+  @Test
+  @DisplayName("nextflow config and run with GLS tutorial")
+  void nextflowFromGLSTutorial() throws IOException, InterruptedException {
+    // select a test user and login
+    TestUsers testUser = TestUsers.chooseTestUserWithSpendAccess();
```

Comment:
It's a tradeoff. It may be more repeatable if we stick with a single test user, but it makes it harder to detect when there's a problem with a test user vs with the test. This came up recently with `william.thunderlord` being in too many Google groups, and I've also accidentally altered permissions of a test user while debugging something and it made tests pass when they shouldn't have. With this random test user approach, we should see flaky tests if a test user gets messed up and hopefully that will help us track down more problems.

Reply:
Added a printout for the test user when we login.